src/utils/form-helper.js

```diff
@@ -10,6 +10,6 @@
  * @property {string} Drivers_License - Drivers License
  */
 
-const docTypes = ['Bank Statement', 'Passport', 'Drivers License'];
+const docTypes = ['Bank Statement', 'Passport', "Driver's License"];
```

Comment:
Makes sense to change it to "Driver's License". But we'll need to update the getContainerUrl in session-helper.

I think we can be done easily by changing Line 169 in session-helper.js which uses

```
case 'Drivers License':
```

to

```
case "Driver's License":
```

This will help us select the correct URL path for the container.